Formula/metals.rb

```diff
@@ -5,6 +5,11 @@ class Metals < Formula
   sha256 "18bcdb08aca2a38409e4af03f1108611285c1d20b9044e5cf2ecd8efa5fc60a2"
   license "Apache-2.0"
 
+  livecheck do
```

Comment:
```suggestion
  # Some version tags don't become a release, so it's necessary to check the
  # GitHub releases instead.
  livecheck do
```

We should add a comment to make it clear that the `GithubLatest` strategy is necessary in this case.